yarn.lock

```diff
@@ -1253,7 +1258,16 @@
     react-vtkjs-viewport "0.0.9"
     vtk.js "^8.11.0"
 
-"@ohif/i18n@0.1.0", "@ohif/i18n@^0.1.0":
+"@ohif/i18n@0.1.1":
+  version "0.1.1"
+  resolved "https://registry.yarnpkg.com/@ohif/i18n/-/i18n-0.1.1.tgz#e4eee433a0a5fb15a41279910ad061e98a371c4c"
+  integrity sha512-ON1UzWDyi2RD79Z+632vvOh82tl6jglzLHZ1M2rQEm5FLP1PQyrwMEBjETA6jMx7mE1XIWQNiGpbln+7AaPSgg==
+  dependencies:
+    "@babel/runtime" "^7.2.0"
+    classnames "^2.2.6"
+    rollup-plugin-json "^4.0.0"
+
+"@ohif/i18n@^0.1.0":
```

Comment:
What is still pulling in `@ohif/i18n@^0.1.0`? `react-viewerbase`?

Reply:
It's not that clear, but I guess it's coming from `react-viewerbase`, since it was not published yet using `@ohif/i18n` `0.1.1`.